static/app/views/automations/components/actionFilters/taggedEvent.tsx

```diff
@@ -33,16 +37,47 @@ function KeyField() {
   const {condition, condition_id, onUpdate} = useDataConditionNodeContext();
   const {removeError} = useAutomationBuilderErrorContext();
 
+  // Select all the tags for an organization to generate a list of the most likely tags
+  const organization = useOrganization();
+  const {data: tagOptions, isLoading} = useFetchOrganizationTags(
+    {
+      orgSlug: organization.slug,
+      projectIds: ['-1'],
+      dataset: Dataset.ERRORS,
+      useCache: true,
+      enabled: true,
+      keepPreviousData: true,
+    },
+    {}
+  );
+
+  if (!tagOptions || isLoading) {
+    return <LoadingIndicator mini size={24} style={{alignItems: 'center'}} />;
```

Comment:
This loading state is a bit jarring since it shifts content. If you add `async` to the select component, I believe there are better loading states